rust/src/write.rs

```diff
@@ -94,7 +94,12 @@ fn write_record<W: Write>(mut w: &mut W, r: &Record) -> io::Result<()> {
         Record::MetadataIndex(mi) => record!(op::METADATA_INDEX, mi),
         Record::SummaryOffset(so) => record!(op::SUMMARY_OFFSET, so),
         Record::DataEnd(eod) => record!(op::DATA_END, eod),
-        _ => todo!(),
+        Record::Unknown { opcode, data } => {
```

Comment:
private?

Reply:
Maybe its' fine as it is, as this covers both private records and unknown future-spec records.

Reply:
Yeah if I did anything I'd split it up into two but I don't really want to break the API just yet.